src/pal/src/configure.cmake

```diff
@@ -903,7 +906,11 @@ else() # Anything else is Linux
   set(KO_KR_LOCALE_NAME ko_KR_LOCALE_NOT_FOUND)
   set(ZH_TW_LOCALE_NAME zh_TW_LOCALE_NOT_FOUND)
   set(HAS_FTRUNCATE_LENGTH_ISSUE 0)
-  set(UNWIND_CONTEXT_IS_UCONTEXT_T 1)
+  if(CMAKE_SYSTEM_PROCESSOR STREQUAL armv7l)
```

Comment:
Can we make a configure time check for this, rather than just bailing on an arch?